### PR TITLE
add: `jellyfin-media-player-deb`

### PR DESCRIPTION
**Pull request by Rockstar9604**

### CODE REVIEW

**Review by Elsie19 — CHANGES_REQUESTED** [review]
Run thru `shfmt`.

**Comment by Rockstar9604** [comment]
> Run thru `shfmt`.

Done :)

File: packages/jellyfin-media-player-deb/jellyfin-media-player-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+"kinetic")
+  url="https://github.com/jellyfin/jellyfin-media-player/releases/download/v${pkgver}/${gives}_${pkgver}-1_amd64-${codename}.deb"
+  hash="ebe627b4db2559f01c7c99f599fc0c5e59097b7ab5e9c168295f513d940a9267"
+  ;;
+*)
+  #Use Bullseye
+  url="https://github.com/jellyfin/jellyfin-media-player/releases/download/v${pkgver}/${gives}_${pkgver}-1_amd64-bullseye.deb"
+  hash="5cac8cdf9271abb14b3fab637d0ef5bc31415198d009417cfaf087cb4218a3cd"
+  ;;
+esac
```
Comment (vigress8) [inline packages/jellyfin-media-player-deb/jellyfin-media-player-deb.pacscript:35]: Use Pacstall's builtin `${DISTRO}` variable, it looks like this `ubuntu:jammy`
```suggestion
codename="${DISTRO#*:}"
echo "Installing for ${codename}"
url="https://github.com/jellyfin/jellyfin-media-player/releases/download/v${pkgver}/${gives}_${pkgver}-1_amd64-${codename}.deb"
case "${codename}" in
  "bookworm")
    hash="2eb9856dc6c7a62c5c635391dee8e5fb35bce06d5c6c132241523e4f04ad2416"
    ;;
  "focal")
    hash="208ead4276fca396d15af7e87df31621cad09fea0a58c3fb33d096febf111a28"
    ;;
  "jammy")
    hash="1fcc64c2d24cf9d9f3c9b29416cbdb921c77bf4497d0be9be92af8b372b96c1b"
    ;;
  "kinetic")
    hash="ebe627b4db2559f01c7c99f599fc0c5e59097b7ab5e9c168295f513d940a9267"
    ;;
  *)
    # Fallback to Bullseye
    url="https://github.com/jellyfin/jellyfin-media-player/releases/download/v${pkgver}/${gives}_${pkgver}-1_amd64-bullseye.deb"
    hash="5cac8cdf9271abb14b3fab637d0ef5bc31415198d009417cfaf087cb4218a3cd"
    ;;
esac
```

File: packages/jellyfin-media-player-deb/jellyfin-media-player-deb.pacscript [hunk 2]
```diff
@@ -0,0 +1,39 @@
+name="jellyfin-media-player-deb"
+gives="jellyfin-media-player"
+pkgver="1.9.1"
+arch=('amd64')
+breaks=("${name}-bin" "${name}" "${name}-app")
+pkgdesc="Jellyfin desktop client using jellyfin-web with embedded MPV player"
+codename="$(lsb_release -cs)"
```
Comment (vigress8) [inline packages/jellyfin-media-player-deb/jellyfin-media-player-deb.pacscript:7]: Use Pacstall's builtin `${DISTRO}` variable, it looks like this `ubuntu:jammy`
```suggestion
codename="${DISTRO#*:}"
```

File: packages/jellyfin-media-player-deb/jellyfin-media-player-deb.pacscript [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  hash="1fcc64c2d24cf9d9f3c9b29416cbdb921c77bf4497d0be9be92af8b372b96c1b"
+  ;;
+"kinetic")
+  url="https://github.com/jellyfin/jellyfin-media-player/releases/download/v${pkgver}/${gives}_${pkgver}-1_amd64-${codename}.deb"
+  hash="ebe627b4db2559f01c7c99f599fc0c5e59097b7ab5e9c168295f513d940a9267"
+  ;;
+*)
+  #Use Bullseye
+  url="https://github.com/jellyfin/jellyfin-media-player/releases/download/v${pkgver}/${gives}_${pkgver}-1_amd64-bullseye.deb"
+  hash="5cac8cdf9271abb14b3fab637d0ef5bc31415198d009417cfaf087cb4218a3cd"
```
Comment (vigress8) [inline packages/jellyfin-media-player-deb/jellyfin-media-player-deb.pacscript:33]: Remove duplicate `url`s.
```suggestion
url="https://github.com/jellyfin/jellyfin-media-player/releases/download/v${pkgver}/${gives}_${pkgver}-1_amd64-${codename}.deb"
case "${codename}" in
"bookworm")
  hash="2eb9856dc6c7a62c5c635391dee8e5fb35bce06d5c6c132241523e4f04ad2416"
  ;;
"focal")
  hash="208ead4276fca396d15af7e87df31621cad09fea0a58c3fb33d096febf111a28"
  ;;
"jammy")
  hash="1fcc64c2d24cf9d9f3c9b29416cbdb921c77bf4497d0be9be92af8b372b96c1b"
  ;;
"kinetic")
  hash="ebe627b4db2559f01c7c99f599fc0c5e59097b7ab5e9c168295f513d940a9267"
  ;;
*)
  #Use Bullseye
  url="https://github.com/jellyfin/jellyfin-media-player/releases/download/v${pkgver}/${gives}_${pkgver}-1_amd64-bullseye.deb"
  hash="5cac8cdf9271abb14b3fab637d0ef5bc31415198d009417cfaf087cb4218a3cd"
```

**Review by vigress8 — CHANGES_REQUESTED** [review]

File: packages/jellyfin-media-player-deb/jellyfin-media-player-deb.pacscript (hunk 2 above)
Comment (Rockstar9604) [reply packages/jellyfin-media-player-deb/jellyfin-media-player-deb.pacscript]: Thank you very much for the advice, I didn't know about the existence of the builtin variable `${DISTRO#*:}`, I had seen `$(lsb_release -cs)` from other pacscripts.

**Comment by Rockstar9604** [comment]
Thanks a lot @vigress8 for all the tips :)

File: packages/jellyfin-media-player-deb/jellyfin-media-player-deb.pacscript [hunk 4]
```diff
@@ -0,0 +1,35 @@
+name="jellyfin-media-player-deb"
+gives="jellyfin-media-player"
+pkgver="1.9.1"
+arch=('amd64')
+breaks=("${name}-bin" "${name}" "${name}-app")
+pkgdesc="Jellyfin desktop client using jellyfin-web with embedded MPV player"
+codename="${DISTRO#*:}"
+url="https://github.com/jellyfin/jellyfin-media-player/releases/download/v${pkgver}/${gives}_${pkgver}-1_amd64-${codename}.deb"
+echo "Installing for ${codename}"
```
Comment (Elsie19) [inline packages/jellyfin-media-player-deb/jellyfin-media-player-deb.pacscript:9]:
```suggestion
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/jellyfin-media-player-deb/jellyfin-media-player-deb.pacscript (hunk 4 above)
Comment (Rockstar9604) [reply packages/jellyfin-media-player-deb/jellyfin-media-player-deb.pacscript]: Done

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**